front50-web/src/main/java/com/netflix/spinnaker/front50/controllers/PipelineController.java

```diff
@@ -202,11 +202,36 @@ public Collection<Pipeline> getTriggeredPipelines(
   public List<Pipeline> listByApplication(
       @PathVariable(value = "application") String application,
       @RequestParam(value = "pipelineNameFilter", required = false) String pipelineNameFilter,
-      @RequestParam(required = false, value = "refresh", defaultValue = "true") boolean refresh) {
+      @RequestParam(required = false, value = "refresh", defaultValue = "true") boolean refresh,
+      @RequestParam(required = false, value = "enabledPipelines", defaultValue = "false")
+          Boolean enabledPipelines) {
```

Comment:
Pretty sure this can be lower-case-b boolean since there's a default.
```suggestion
          boolean enabledPipelines) {
```